mvc/models/api.models.js

```diff
@@ -0,0 +1,5 @@
+const endpoints = require('../../endpoints.json')
+
+exports.fetchEndpoints = () => {
```

Comment:
We don't actually need a model here, we can just require the endpoints into the controllers file and send it from there 🙂